grails-app/jobs/mama/ng/scheduler/SchedulerJob.groovy

```diff
@@ -0,0 +1,91 @@
+package mama.ng.scheduler
+
+import groovy.time.TimeCategory
+import groovyx.net.http.ContentEncoding
+import groovyx.net.http.ContentType
+import groovyx.net.http.HTTPBuilder
+import groovyx.net.http.HttpResponseException
+import groovyx.net.http.Method
+
+class SchedulerJob {
+    def concurrent = false
+
+    def cronParserService
+
+    static triggers = {
+        cron name:'cronTrigger', startDelay:1000, cronExpression: '0 0 * * * ?' } //execute every hour on the hour
+
+    def group = "Scheduler"
+
+    def execute() {
+        use (TimeCategory) {
+            def then = new Date() + 1.hour
+            def schedules = Schedule.findAllByNextSendLessThanEquals(then)
+
+            schedules.each { Schedule schedule ->
```

Comment:
Could we split ``execute`` up into two methods -- one for schedules and the other for messages? It'll make the method shorter and I think the logic clearer.